mcs/class/Microsoft.Build.Engine/Test/Microsoft.Build.BuildEngine/ProjectTest.cs

```diff
@@ -1301,6 +1301,34 @@ public void TestBuildProjectFileXml8 ()
 		}
 
 		[Test]
+		public void TestBatchedMetadataEmptyBatch () {
```

Comment:
PR == pull request (this patch). The test passes even on Mono master, so it doesn't verify the fix.